instrumentation/instaamqp/README.md

```diff
@@ -0,0 +1,119 @@
+Instana instrumentation for streadway-amqp
+==========================================
+
+This module contains instrumentation code for RabbitMQ clients written with [`streadway-amqp`](https://pkg.go.dev/github.com/streadway/amqp@v1.0.0).
+
+[![GoDoc](https://pkg.go.dev/badge/github.com/instana/go-sensor/instrumentation/instaamqp)][godoc]
+
+Installation
+------------
+
+To add the module to your `go.mod` file run the following command in your project directory:
+
+```bash
+$ go get github.com/instana/go-sensor/instrumentation/instaamqp
+```
+
+Usage
+-----
+
+`instaamqp` offers a function wrapper around [`amqp.Channel`][amqp.Channel] that returns an `instaamqp.AmqpChannel` instance.
+This Instana object provides instrumentation for the `amqp.Channel.Publish` and `amqp.Channel.Consume` methods, that are
+responsible for tracing data from messages sent and received.
+
+For any other `amqp.Channel` methods, the original `amqp.Channel` instance can be normally used.
+
+A publisher example:
+
+```go
+func Example_publisher() {
```

Comment:
It solves the issue of a potential human error in case the example is updated only in one place later on.
On the other hand, my opinion is that removing an example from a README file defeats the purpose of having one. The example is part of the document, so users don't have to navigate back and forth to follow the docs and read the code at the same time.